fitbenchmarking/fitting/controllers/ralfit_controller.py

```diff
@@ -4,6 +4,7 @@
 """
 
 import ral_nlls
+import sys
```

Comment:
I can't see where sys is used, is there a reason for importing it?

Reply:
Yes -- thanks!  It was left over from the exception handling that I took out yesterday.